clib.json

```diff
@@ -1,8 +1,14 @@
 {
   "name": "nuklear",
-  "version": "4.09.0",
+  "version": "4.10.0",
   "repo": "Immediate-Mode-UI/Nuklear",
   "description": "A small ANSI C gui toolkit",
-  "keywords": ["gl", "ui", "toolkit"],
-  "src": ["nuklear.h"]
+  "keywords": [
+    "gl",
+    "ui",
+    "toolkit"
+  ],
+  "src": [
+    "nuklear.h"
+  ]
 }
```

Comment:
Let's not change the formatting of the `clib.json`. Only bump the version

Reply:
Any reason? To me it seems easier to read this way

Reply:
I'm indifferent here. But generally speaking if such changes shall further extend big PRs I'd beg everyone to leave them out for the sake of clarity.

Reply:
I would personally also like it to be formatted this way, as then it would be inline with my json formatter (4 spaces would be nice too). If enough people agree, then sure we can change it, but I don't want to end up en a situation where we change the clib.json format in every PR, if you know what I mean :)